jme3-core/src/main/java/com/jme3/app/DetailedProfilerState.java

```diff
@@ -413,7 +424,8 @@ public void layout(int indent) {
             int y = -(height * LINE_HEIGHT + HEADER_HEIGHT);
 
             label.setLocalTranslation(PADDING + indent * PADDING, y, 0);
-            float gpuPos = PANEL_WIDTH - font.getLineWidth(gpuText.getText()) - PADDING * (maxLevel - indent + 1);
+            float gpuPos = PANEL_WIDTH - font.getLineWidth(gpuText.getText()) - PADDING
```

Comment:
I suggest breaking the line before `- PADDING` so that the final term is not split across 2 lines.